developerportal/templates/molecules/cards/card.html

```diff
@@ -5,5 +5,5 @@
 {% elif resource.resource_type == "person" %}
   {% include "molecules/cards/card-person.html" with resource=resource filter_target=filter_target %}
 {% elif resource.resource_type == "video" %}
-  {% include "molecules/cards/card-video.html" with resource=resource %}
+  {% include "molecules/cards/card-video.html" with resource=resource filter_target=filter_target %}
```

Comment:
I don't thikn adding this extra param is dangerous (re other uses of `card-video.html`) because if `filter_target` is not present, that counts as falsey in Django templates, so the `{% if filter_target %}` blocks added to card-`video.html` will go unevaluated

Reply:
I still think this 😄